PSIni/Functions/Out-IniFile.ps1

```diff
@@ -194,13 +200,13 @@ Function Out-IniFile {
                 Add-Content -Value "$i$delimiter$($InputObject[$i])" @parameters
 
             }
-            elseif ($i -eq $script:NoSection) {
-                #Key value pair of NoSection
-                Out-Keys $InputObject[$i] `
-                    @parameters `
-                    -Delimiter $delimiter `
-                    -MyInvocation $MyInvocation
-            }
+#            elseif ($i -eq $script:NoSection) {
+#                #Key value pair of NoSection
```

Comment:
Why are you commenting this part out?

Reply:
Uh, I am deeply sorry, about this. This is a remained of a debugging step.